### PR TITLE
Changes by create-pull-request action

### PR DESCRIPTION
**Pull request by Starz0r**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

**Review by WhippuSIF — CHANGES_REQUESTED** [review]
Remove the link for the status reference. That was also a mistake on my part.

Both this and #183 will resolve [issue #1686](https://github.com/AreWeAntiCheatYet/AreWeAntiCheatYet/issues/1686) once revisions are made.

File: manifest-new.json [hunk 1]
```diff
@@ -8305,5 +8305,21 @@
...
     "dateChanged": "2024-07-25T21:21:20.558Z"
+  },
+  {
+    "url": "https://www.cbjq.com/",
+    "slug": "snowbreak-containment-zone-china",
+    "name": "Snowbreak: Containment Zone (CHN)",
+    "logo": "",
+    "native": false,
+    "status": "Broken",
+    "reference": "https://www.cbjq.com/show-7126-3-1.html",
```
Comment (WhippuSIF) [inline manifest-new.json:8316]: Should be left blank

**Comment by Starz0r** [comment]
Hey, I wanted to thank you for these PRs, even if I'm not able to merge them. So far you are the only other person besides me that knows about the editor!

As you can see, it's still a work-in-progress, but I still try to do my day to day work in it. It still has a few issues, including one it seems you ran into, like being unable to add new anti-cheats. In a way, you have to add it out-of-band first before it shows up, which is an issue I'd like to fix later.

I'm glad you at least attempted these changes at all! Unfortunately, I can't exactly accept these changes as-is at the moment, and this isn't your fault, this is just a duck-taped system I have at the moment to make my life easier. From here, I'll most likely be closing this, and taking the changes and submitting them again so they're mergable. I do plan to allow the editor to be used publicly, and accept submissions from it, but it just isn't there yet. Nonetheless, thank you.

**Comment by WhippuSIF** [comment]
@Starz0r Thanks. I'll keep that in mind.

**Pull request closed without merging**